maro/cli/process/deployment/process_setting.yml

```diff
@@ -0,0 +1,7 @@
+redis_info:
```

Comment:
Since you load default configurations from default_params.yml, do you think it's necessary to keep this file?

Reply:
Actually, this file is used for customized setting deployment. I am going to rename it.